btree/btree.c

```diff
@@ -2422,21 +2360,18 @@ static bool ff_invariant(const struct nd *node)
 {
 	const struct ff_head *h = ff_data(node);
 
-	return  _0C(h->ff_shift == segaddr_shift(&node->n_addr)) &&
-		_0C(node->n_skip_rec_count_check ||
-		    ergo(h->ff_level > 0, h->ff_used > 0));
+	return  _0C(h->ff_shift == segaddr_shift(&node->n_addr));
```

Comment:
Is this the final definition of ff_invariant

Reply:
No, This was just done to accomodate segaddr changes, but for now I kept them as it is.
In the future jira, that you will work on, will have the final definition of ff_invariant.

Reply:
ok